src/NativeScript/GlobalObject.mm

```diff
@@ -7,45 +7,46 @@
 //
 
 #include "GlobalObject.h"
-#include <string>
```

Comment:
clang format on my machine rearranged those but if you'd like I will bring them back as they used to be 🌹